### DISCUSSION TITLE
Is there any guide to Create a Nuxt 3 project manually?

### DISCUSSION THREAD

**Discussion by alperunal** [nuxt/nuxt#17221] [Questions] [answered]
Hi, I've checked the documentation but couldn't find anything about this.
Is there any way to install Nuxt 3 manually instead of creating a new project using the cli?

**Comment by TheAlexLichter** [comment, marked as answer]
Yes! All you need is a simple `package.json`, installing the dependencies via your favorite package manager and then you can drop in your `App.vue` to get started (or use the `pages` folder instead):

```json
{
  "name": "your-project",
  "private": true,
  "devDependencies": {
    "nuxt": "npm:nuxt3@latest"
  },
  "scripts": {
    "dev": "nuxi dev",
    "build": "nuxi build",
    "start": "nuxi preview"
  }
}
```

**Reply by alperunal** [reply]
@manniL Thank you for your answer. 
I noticed where is the problem on my local thanks to your answer:
When I install as, `npm i nuxt3@latest --save-dev` it saves to devDependencies as "nuxt3" then when I try to run dev, it can't find nuxt.
